charts/postcoder-proxy/values.yaml

```diff
@@ -13,15 +13,16 @@ externalSecrets: &external_secrets_config
   stores: {}
 
 env:
-  APP_ENV: production
+  APP_ENV: development
```

Comment:
If you make changes to the default values, you need to increment the chart version